packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1656,8 +1656,8 @@ class ArmBuildImage implements IBuildImage {
   public validate(buildEnvironment: BuildEnvironment): string[] {
     const ret = [];
     if (buildEnvironment.computeType &&
-        buildEnvironment.computeType !== ComputeType.LARGE) {
-      ret.push(`ARM images only support ComputeType '${ComputeType.LARGE}' - ` +
+        buildEnvironment.computeType !== (ComputeType.SMALL || ComputeType.LARGE)) {
```

Comment:
I don't think this makes sense... `ComputeType` is at runtime a string, so `ComputeType.SMALL || ComputeType.LARGE` just returns `ComputeType.SMALL`, so this will not allow you to specify `computeType` as `ComputeType.LARGE`.

This should be just `buildEnvironment.computeType !== ComputeType.SMALL && buildEnvironment.computeType !== ComputeType.LARGE`.